src/main/java/com/afornalik/ox/model/board/OverrideFieldException.java

```diff
@@ -0,0 +1,7 @@
+package com.afornalik.ox.model.board;
+
+public class OverrideFieldException extends Exception {
```

Comment:
Like I said before, I am not convinced that using exceptions to validate input is a good solution. Maybe think about something else?

Reply:
I too rashly create exception - i will change it soon.